app/dashboard/templates/github_comments/issue_opened_marketing_footer.md

```diff
@@ -0,0 +1,5 @@
+{% load i18n static humanize site %}
```

Comment:
Can we remove unused loads throughout all of the md files?

Reply:
@mbeacom  done!